frontend/src/pages/CreateTask.js

```diff
@@ -0,0 +1,516 @@
+                        value={ipfsDetail}
+                        onChange={this.writeIPFSDetail}
+                      />
+                    </div>
+                    <button className="button" type='submit'>
+                      Submit
+                    </button>
+                  </form>
+              </div>
+            }
+          </div>
+          <div className="task-create-column task-preview">
+            <h2>Task Preview</h2>
+            <p>Note that <b>this task insert costs gas</b>, so we show you this preview here.  Make sure it's valid and as you want.</p>
+            <div className="task-preview-content">
+              <p className="inline">struct</p>
+              <span className="word-separator">
+                Task</span>
+              &#123;
+              <div className="struct-line">
+                <span className="task-preview-key">
+                title:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': titlePrepared })}>
+                  {titlePrepared}
+                </span>
+              </div>
+              <div className="struct-line">
+                <span className="task-preview-key">
+                  project:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': project })}>
+                  {project}
+                </span>
+              </div>
+              <div className="struct-line">
+                <span className="task-preview-key">
+                  subProject:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': subProject })}>
+                  {subProject}
+                </span>
+              </div>
+              <div className="struct-line">
+                <span className="task-preview-key">
+                  createdBy:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': account })}>
+                  {account}
+                </span>
+              </div>
+              <div className="struct-line">
+                <span className="task-preview-key">
+                  ipfsHash:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': ipfsHash })}>
+                  {ipfsHash}
+                  </span>
+              </div>
+              <div className="struct-line">
+                <span className="task-preview-key">
+                  url:
+                </span>
+                <span className={classNames('task-preview-value', { 'bg-light-gray': url })}>
+                  {url}
+                </span>
+              </div>
+              }
+            </div>
+          </div>
+        </div>
+        <style jsx>{`
+          .task-create-view {
+            display: -ms-flex;
+	          display: -webkit-flex;
```

Comment:
So this just becomes `display: flex;` and that's it, right?  This is for all styles?  Didn't find a ton about this by googling